### PR TITLE
Install kafka and postgres from operatorHub

### PR DESCRIPTION
**Pull request by clyang82**
ref: https://issues.redhat.com/browse/ACM-7166

### CODE REVIEW

File: operator/pkg/controllers/hubofhubs/globalhub_controller.go [hunk 1]
```diff
@@ -188,6 +192,50 @@ func (r *MulticlusterGlobalHubReconciler) reconcileGlobalHub(ctx context.Context
...
+
+		if err := r.EnsureKafka(ctx); err != nil {
+			return err
+		}
+
+		r.MiddlewareConfig.KafkaConnection, err = r.WaitForKafkaClusterReady(ctx)
+		if err != nil {
+			return err
+		}
+	}
```
Comment (yanmxa) [inline operator/pkg/controllers/hubofhubs/globalhub_controller.go:238]: How to uninstall the kafka and postgres by the operator?

Comment (clyang82) [reply operator/pkg/controllers/hubofhubs/globalhub_controller.go:238]: Will not uninstall kafka and postgres

File: operator/main.go [hunk 2]
```diff
@@ -243,9 +258,7 @@ func buildResourceFilterMap() cache.NewCacheFunc {
 			&corev1.Service{}: {
 				Label: labelSelector,
 			},
-			&appsv1.Deployment{}: {
-				Label: labelSelector,
-			},
+			&appsv1.Deployment{}: {},
```
Comment (yanmxa) [inline operator/main.go:261]: Should we add a label to cache the specific deployment?

**Pull request merged**